internal/driver/glfw/window.go

```diff
@@ -504,24 +504,29 @@ func (w *window) findObjectAtPositionMatching(canvas *glCanvas, mouse fyne.Posit
 	return driver.FindObjectAtPositionMatching(mouse, matches, canvas.Overlays().Top(), roots...)
 }
 
+func cursorToName(cursor desktop.Cursor) *glfw.Cursor {
+	var cursorMap = map[desktop.Cursor]*glfw.Cursor{
```

Comment:
Doesn't this create a new map on every call?

Reply:
Yes, it does but that shouldn't be an issue. We need to reduce the amount of global variables.

Reply:
Why?

Reply:
Yes, this could be called a lot and I don't see the harm in a single map global to the package that is set up in init()

Reply:
Also I have noticed this method name is strange. Maybe it should be "nativeCursor" or "lookupNativeCursor"?